magefile.go

```diff
@@ -139,10 +146,11 @@ func getMixins() error {
 
 	mixins := []struct {
 		name    string
+		url     string
 		feed    string
 		version string
 	}{
-		{name: "helm3", feed: "https://mchorfa.github.io/porter-helm3/atom.xml", version: "v0.1.14"},
+		{name: "helm3", url: "https://github.com/carolynvs/porter-helm3/releases/download", version: "v0.1.15-8-g864f450"},
```

Comment:
This is a build of helm3 that includes a patch for running in a nonroot bundle. https://github.com/MChorfa/porter-helm3/pull/42 I'll submit that PR when we've fully vetted the nonroot change.